src/utils/encoreLogOutTimer.js

```diff
@@ -63,8 +65,30 @@ function EncoreLogOutTimer() {
       utils.deleteCookie('PAT_LOGGED_IN');
       utils.deleteCookie('ENCORE_LAST_VISITED');
       utils.deleteCookie('nyplIdentityPatron');
+      this.loadLogoutIframe();
     }, timeTillLogOut);
   };
+
+  /**
+   * loadLogoutIframe()
+   * The function that loads a temporary iframe with the log out endpoint
+   * to completely log out the user from Encore. It then deletes the iframe right away.
+   * The reason to use this way to load the endpoint is to bypass the CORS loading from the client
+   * since III does not want to provide us a real log out API URI.
+   */
+  this.loadLogoutIframe = () => {
+    const logoutIframe = document.createElement('iframe');
+    const body = document.getElementsByTagName('body')[0];
+
+    logoutIframe.setAttribute(
+      // The endpoint is the URL for logging out from Encore
+      'src', 'https://browse.nypl.org/iii/encore/logoutFilterRedirect?suite=def'
+    );
+    // Assigns the ID for CSS ussage
+    logoutIframe.setAttribute('id', 'logoutIframe');
+    body.appendChild(logoutIframe);
+    setTimeout(() => { body.removeChild(logoutIframe); }, 100);
```

Comment:
We can definitely make the time longer. I just tried to reduce the duration the time iframe exists. How do you think 500 millisecond?

Reply:
Longer is better, but is there a reason to not extend it to the longest we can reasonably expect someone on a slow connection to load the Logout page? It's hidden to the user, so is there any detriment to leaving it in the page?

Reply:
You mean just to leave the iframe there? I guess I want to delete it after its job is done since we don't really need it. And I am afraid having multiple iframe with with browse.nypl.org will cause us issues if the user sign in again.

Reply:
For argument's sake, yes I think we could leave it there. The issue I see removing it early is we don't really know when it's job is done. It will *probably* be done in 100ms, but if Sierra is slow or user is on a slow connection or has a wifi hiccup, leaving the iframe on the page for a bit increases the chance it does its job without hindering user experience.

Reply:
Do you suggest that we just load it and never delete it? Or we still delete it but with a longer waiting time.

Reply:
Per offline discussion, 10s sounds better. Thanks!